.eleventy.js

```diff
@@ -297,7 +297,9 @@ module.exports = function(eleventyConfig) {
                 const commonTags = requiredTags?.reduce((count, tag) => count + (post.data.tags?.includes(tag) ? 1 : 0), 0);
                 return { ...post, commonTags };
             })
-            .filter(post => post.url !== page.url && post.commonTags >= requiredTags.length - 1);
+            .filter(post => post.url !== page.url && post.commonTags >= requiredTags.length - 1)
+            .sort((a, b) => b.commonTags - a.commonTags || b.date - a.date)
```

Comment:
This must be JS magic, but when is `b.commonTags - a.commonTags` falsy?

Reply:
This seems like magic to me as well. It's subtracting the amount of tags in common of two posts at a time to determine the order, if two posts have the same amount of tags, the result is 0 (falsy), and then it compares by date and puts the most recent first.